src/api/app/views/webui/user/user_profile_redesign/_involvement.html.haml

```diff
@@ -0,0 +1,71 @@
+- roles = Role.local_roles
+#involvement
+  .card.sticky-top#involved-filters
+    .card-body
+      = form_with(model: displayed_user, method: :get, local: true, class: 'form-inline justify-content-between') do
+        .form-group
+          %input.form-control{ 'aria-label': 'Search', name: 'search_text', placeholder: 'Search', type: 'text', value: filters[:search_text] }
+        %span.dropdown.form-group#involved-project-package
+          %button.btn.btn-outline-secondary.dropdown-toggle.form-control{ data: { toggle: :dropdown }, type: :button }
+            %span.caret
```

Comment:
probably an option to reset the filter is necessary?

Reply:
Do you mean like options `All` and `None` used in the filters in the project monitor page?

![Screenshot from 2020-10-16 12-17-26](https://user-images.githubusercontent.com/24919/96246900-9ba36b00-0fa9-11eb-8052-08354b75710f.png)

We decided not to include them in the initial mockup, because there were not so many items in the filter.

Reply:
I'm talking about "Involved Package and Projects"

Once the filter is applied by the user then a button called ‘Reset Filters’ or ‘’Clear All’’ should be given next to the filters so that the user can quickly make changes. If we don't have it, we will force the user to either refresh the page (assuming that the filter will be erased) or user have to empty the search and click on "Filter results"

 Github does the same:

![Screenshot_2020-10-16_12-35-48](https://user-images.githubusercontent.com/37418/96248634-25543800-0fac-11eb-8bca-f58cc84a0e5a.png)

Reply:
Now I get it. A button called ‘Reset Filters’ or ‘’Clear All’’ was not included in the initial mockup. Probably something we forgot to add.